templates/complete_table_tasks.plush.html

```diff
@@ -53,11 +53,11 @@
                 <td class="text-center"> <%= linkTo( uncheckPath ({ task_id: task.ID }), {class: "fa-regular fa-circle-xmark fs-5 text-dark text-decoration-none" }) %></td>
                 <td class="text-center">
                     <%= linkTo(tableCompletePath({task_id: task.ID}), {class: "text-decoration-none text-dark"}) { %>
-                        <%= task.Name_task %>
+                        <%= task.Name %>
                         <% } %>
                 </td>
                 <td class="text-center"><%= task.CreatedAt.Day() %> <%=  task.CreatedAt.Month() %> <%=  task.CreatedAt.Year() %> </td>
-                <td class="text-center"><%= task.Finish_at.Format(Format(tsdfsdfsdf) %> </td>
+                <td class="text-center"><%= task.FinishedAt.Time.Day() %> <%= task.FinishedAt.Time.Month() %> <%= task.FinishedAt.Time.Year() %></td>
```

Comment:
It looks ok, but you can get that time with the correct format in a simpler way. Try to use "Format" method.